### DISCUSSION TITLE
Rename zip file

### DISCUSSION THREAD

**Discussion by dark0dev** [SamDel/ChromeCast-Desktop-Audio-Streamer#151] [Ideas]
How about renaming file "Setup 5.3.zip" to something more meaningfull like "ChromecastDesktopAudioStreamer-5.3.zip"?

**Comment by SamDel** [comment]
👍 
I will change it in the next release. Hope I don't forget because it's done manually. Maybe I add a build event for it.

**Comment by SamDel** [comment]
This is implemented in the [5.4 release](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/releases/tag/v5.4).
